scripts/run_backend_tests.py

```diff
@@ -149,7 +148,9 @@ def run_shell_cmd(exe, stdout=subprocess.PIPE, stderr=subprocess.PIPE):
     If the cmd fails, raises Exception. Otherwise, returns a string containing
     the concatenation of the stdout and stderr logs.
     """
-    p = subprocess.Popen(exe, stdout=stdout, stderr=stderr)
+    python_path = ':'.join(sys.path)[1:]
+    p = subprocess.Popen(
+        exe, stdout=stdout, stderr=stderr, env={'PYTHONPATH': python_path})
```

Comment:
I feel this is causing failure in the CI test. Only one environment variable is passed to child procedure while the tests may need other ones. Especially `common.py` depends on the `PATH` environment variable. https://github.com/oppia/oppia/blob/a99e0955d53abb74e29ae9e446a5de8a3a72ba73/scripts/common.py#L43

https://github.com/cuichenli/oppia/blob/e44dafeeb7c5f865153d90b4553947503c0e316b/scripts/run_backend_tests.py#L159
This is the way I used to fix the `coverage` problem, you may want to have a look.

Reply:
Hmm, I'm still getting issues in the backend, can you ptal? https://github.com/kevinlee12/oppia/pull/109

Reply:
Had a quick look, you may miss this line 
https://github.com/cuichenli/oppia/blob/e44dafeeb7c5f865153d90b4553947503c0e316b/scripts/run_backend_tests.py#L355

Reply:
Hmm, isn't that this though? https://github.com/kevinlee12/oppia/pull/109/files#diff-f5c2de7b0561fece76aa9264f0192678R361

Reply:
Ah right, the path should be directory instead of the binary file. 

https://github.com/cuichenli/oppia/blob/e44dafeeb7c5f865153d90b4553947503c0e316b/scripts/run_backend_tests.py#L357

Reply:
Ah, I see what went wrong. I needed to add the path to `coverage-4.5.4` to the `PYTHONPATH`, but executing coverage means that I have to go to `coverage-4.5.4/coverage` (which itself is also a path). Thanks for your help @cuichenli!